apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -471,44 +471,31 @@ export const getMultipleChoiceSectionSummary = (state) => {
 /**
  * Takes in an array of objects {answerText: '', correct: true/false} and
  * returns the corresponding letters to the options with the correct answers.
- *
- * TODO(caleybrock): Add letter options to response from the server so they are
- * consistent with the structure, but for now look up letter in this array.
- * If this code is left client side, it needs tests.
- *
  * Ex - [{correct: false}, {correct: true}] --> returns 'B'
  */
-const getCorrectAnswer = (answerArr) => {
+export const getCorrectAnswer = (answerArr) => {
   if (!answerArr) {
     return '';
   }
-
-  const letterArr = ['A','B', 'C', 'D', 'E', 'F', 'G', 'H'];
   const correctLetters = [];
   for (let i = 0; i < answerArr.length; i++) {
     if (answerArr[i].correct) {
-      correctLetters.push(letterArr[i]);
+      correctLetters.push(ANSWER_LETTERS[i]);
     }
   }
   return correctLetters.join(', ');
```

Comment:
Could implement this function as:
```js
const correctIndexes = answerArr
  .map(({correct}, i) => (correct ? i : null))
  .filter(i => i !== null);
return return indexesToAnswerString(correctIndexes);
```
...but that might not be more readable.  Up to you.